### ISSUE TITLE
Claude 100k token options?

### ISSUE THREAD

**Issue by nick-inkeep** [nat/openplayground#85]
See nat.dev has the 100k variations of claude but this repo doesn't. Can they be added?

**Comment by Spshulem** [comment]
Yeah this seems like a dead repo - only the website is getting updates
